nzedb/Binaries.php

```diff
@@ -724,27 +748,33 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 					// Get the current unixtime from PHP.
 					$now = time();
 
+					$xref = ($multiGroup === true ? sprintf('xref = CONCAT(xref, "\\n"%s ),', $this->_pdo->escapeString(substr($header['Xref'], 2, 255))) : '');
```

Comment:
if it is done soley for NZB creation, AND it's done at the same time, then may as well create the array/list in RAM. If on the other hand it is done ad hoc (whenever a new group is discovered during header processing) then longer term storage is need: hence table or column